packages/client/src/runtime/core/errorRendering/applyValidationError.ts

```diff
@@ -439,6 +486,22 @@ function addSelectionSuggestions(selection: ObjectValue, outputType: OutputTypeD
   }
 }
 
+function addInclusionSuggestions(selection: ObjectValue, outputType: OutputTypeDescription) {
+  for (const field of outputType.fields) {
+    if (field.isRelation && !selection.hasField(field.name)) {
+      selection.addSuggestion(new ObjectFieldSuggestion(field.name, 'true'))
+    }
+  }
+}
+
+function addExclusionSuggestions(selection: ObjectValue, outputType: OutputTypeDescription) {
+  for (const field of outputType.fields) {
+    if (!selection.hasField(field.name) && !field.isRelation) {
+      selection.addSuggestion(new ObjectFieldSuggestion(field.name, 'true'))
+    }
+  }
+}
+
```

Comment:
Are these for the typescript language-server or am I misunderstanding what `suggestion` means